### PR TITLE
Update IRMA to 1.1.2

### PR DESCRIPTION
**Pull request by jwarnn**
Hello,
IRMA just recently had an update. In the update they fixed issue with SRR reads so those comments have been removed from the README and associated code removed from the test layer. The test layer runs both the test_run.sh that is included with IRMA and some test fastq files from an online database. As part of the update CDC also posted an image of IRMA 1.1.2 on their dockerhub; is this an issue having competing images? 

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by kapsakcj** [comment]
Thanks for the PR. I can review sometime next week unless someone beats me to it

> is this an issue having competing images?

No, it should not be as the code is open-source and licensed as such (IIRC).

It would help to note in the `irma/1.1.2/README.md` that this docker image is not maintained by CDC & the IRMA developers, so if users encounter issues with this docker image specifically they should file an issue on this github repo and do not contact CDC for support. We can't expect CDC to provide support for this "3rd party" docker image

**Comment by jwarnn** [comment]
Added note to the README about the CDC image and who to contact for support.

**Comment by erinyoung** [comment]
This looks great! Can you add this version of IRMA to the main readme, too?

**Comment by jwarnn** [comment]
Added version to README

**Comment by jwarnn** [comment]
This was on the StaPH-B slack:

"[@Curtis (Theiagen)](https://staph-b-dev.slack.com/team/UBX2WRMU0)
 There was a bug/regression in v1.1.2 that I have fixed. Recommend using v1.1.3 that was released today. See: [Dockerhub](https://hub.docker.com/r/cdcgov/irma) and [CDC Wonder](https://wonder.cdc.gov/amd/flu/irma/) (edited)"
 
 Should I just update to v1.1.3 and forget about v1.1.2?

**Comment by erinyoung** [comment]
@jwarnn , adjusting your current dockerfile is fine, or you can add a new Dockerfile for the new version (both would be sent to Dockerhub/quay, but only 1.1.3 would get the 'latest' tag).

**Comment by kapsakcj** [comment]
Yes, I would advocate for skipping 1.1.2 and going right to 1.1.3

**Comment by jwarnn** [comment]
I added a 1.1.3 dockerfile to go along with what I had already added.

File: irma/1.1.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+# WORKDIR sets working directory
+WORKDIR /data
+
+FROM app as test
+
+RUN cd /flu-amd/tests && \
+  ./test_run.sh
+
+RUN wget ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR179/072/SRR17940172/SRR17940172_1.fastq.gz && gzip -d SRR17940172_1.fastq.gz && \
```
Comment (erinyoung) [inline irma/1.1.3/Dockerfile:61]: According to https://wonder.cdc.gov/amd/flu/irma/run.html, irma can now be run on fastq.gz files. What happens if you remove the gzip -d?

**Review by erinyoung — APPROVED** [review]
I approve this PR. 

What do you think about changing the tests to remove the decompression step?

Changes that I made:
- added hyperlink to the main README.md
- changed the name of the irma 1.1.3 README.md file to README.md
- formatting in the 1.1.2 README.md

**Comment by jwarnn** [comment]
It ran fine. I could also take out for the apt-get step git and gzip.

**Comment by erinyoung** [comment]
Thanks!

I'm going to 
1. merge this PR
2. push to dockerhub and quay for both Dockerfiles using 1.1.2 and 1.1.3
3. overwrite the 'latest' tag for 1.1.3

**Comment by erinyoung** [comment]
Thank you for your contribution!

You can check the status of your deploy at https://github.com/StaPH-B/docker-builds/actions/runs/6814733115 and https://github.com/StaPH-B/docker-builds/actions/runs/6814737800

**Pull request merged**